tests/sentry/auth/test_superuser.py

```diff
@@ -71,10 +72,19 @@ def build_request(
     def test_ips(self):
         user = User(is_superuser=True)
         request = self.make_request(user=user)
+        request._body = json.dumps(
+            {
+                "superuserAccessCategory": "Edit organization settings",
+                "superuserReason": "Edit organization settings",
+            }
+        )
```

Comment:
Doesn't `make_request` now do this with `is_superuser=True`? Is there some reason I'm missing that we don't want to call `make_request` that way?

Reply:
It only does that if `is_superuser=True` is passed into `make_request` but unfortunately there are unwanted behaviours that `make_request` does if `is_superuser=True` there are examples of other tests that test superuser functionalities that call `make_request` without `is_superuser=True` to avoid the unwanted behaviours